cocos/2d/CCActionManager.cpp

```diff
@@ -315,6 +315,37 @@ void ActionManager::removeAllActionsByTag(int tag, Node *target)
     }
 }
 
+void ActionManager::removeActionsByFlags(unsigned int flags, Node *target)
+{
+    if (flags == 0)
+    {
+        return;
+    }
+    CCASSERT(target != nullptr, "");
```

Comment:
I totally agree. 

Actually this is c&p-code from `ActionManager::removeAllActionsByTag()`. 
As a matter of fact, there are a several "empty" asserts of this kind in CCActionManager.cpp (same in CCActionInterval.cpp, CCActionEase.cpp, CCScheduler.cpp...)

Maybe someone should have a look at all of them and change them in one commit...